### ISSUE TITLE
chore(deps): clsx unexpected version

### ISSUE THREAD

**Issue by johannpinson** [joe-bell/cva#317]
**Describe the bug**
Hello,
The "clsx" deps is actually fixed to the v2.0.0.
The actual code was updated in april to change it to v2.1.0 (not released on npm), but it's not allowed to use another minor/patch version. So we cannot used the latest v2.1.1 to follow changes made in  `@types/react`.

**To Reproduce**
In a monorepo config, if we force the installed version in a workspace, we can conflict with the version expected by CVA, example:

```
Type 'import("/Users/xxx/packages/ui/node_modules/clsx/clsx", { with: { "resolution-mode": "import" } }).ClassValue'
is not assignable to type 'import("/Users/xxxx/node_modules/clsx/clsx").ClassValue'.
  Type 'bigint' is not assignable to type 'ClassValue'.ts(2322)
```

**Expected behavior**
In the pkg.json, the dependencies may be set as `"^2.1.0"` instead of the fixed version `"2.1.0"`, and be released as a patch update (instead of waiting for the v1 release).

Thanks for our work!

**Comment by joe-bell** [comment]
Fixed by #316

**Comment by johannpinson** [comment]
Ok perfect, I didn't see this pending PR, I have checked only issues to avoid a duplicate request.
